### PR TITLE
Rewrite + switch to Quarto

### PR DESCRIPTION
**Pull request by ekiefl**
This PR follows in the wake of https://github.com/Arcadia-Science/2024-paired-token-masking/pull/1

# Content changes

After hearing input/advice of several people, I decided to shorten the publication by focusing on two main aspects:

* I shortened the introductory material, so it reads like less of a half-baked tutorial and more like useful background information.
* I removed the section about "unilaterally affecting residues". I originally added this section because it seemed like the next logical step to take the analysis, but after stepping away for nearly a month, I found it to be overly speculative and just one of many directions a followup analysis could go. Chiefly, the purpose of this pub is to get feedback on why the epistatic contact map pattern diminishes with larger model sizes, and the unilaterally affecting residue analysis isn't required to ask why this occurs.

In addressing the above two points, I ended up rewriting much of the pub.

# Infrastructure changes

In https://github.com/Arcadia-Science/2024-paired-token-masking/pull/1, the notebook was rendered in a doc-style format with nbsphinx. This is a bit clunky, and I decided to instead render with Quarto, which offers really nice rendering options like full width figures, margin content, figure captioning, and more.

Any feedback about quarto infrastructure will be addressed in the notebook pub template repo (and subsequently applied to this repo), found here: https://github.com/Arcadia-Science/notebook-pub-template

# Viewing the publication

The publication is currently hosted at https://arcadia-science.github.io/2024-paired-token-masking/

# Misc

* The logits arrays were incorrectly calculated because of `CLS` token indexing that I wasn't aware of. These have been recalculated correctly.
* Please no feedback on beautification, that is being handled within the Publishing team.

### CODE REVIEW

**Review by braebigge — APPROVED** [review]
I added a few comments, but I think overall it's great! The rewrite is more clear and I found it pretty easy to follow. I'm excited for the next steps and to see what pub team does with it!

**Comment by keithchev** [comment]
<p>+1 I'd explain what "reconstruction accuracy" refers to (is it just how often the model predicts the correct residue?)</p>

---

 <a href='https://app.reviewnb.com/Arcadia-Science/2024-paired-token-masking/pull/4/discussion/#comment-2568255049'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by keithchev** [comment]
<p>Yeah, agreed. If the main goal of the pub is specifically to pose this question to external readers (rather than just generally share a surprising result), I'd highlight or mention this question in the introduction, and maybe even summarize the result and question to excite the reader's curiosity (s/t like "we found that a certain measure of residue-pair model uncertainty mimics a physical contact map, and we're wondering if you can tell us why!")</p>

---

 <a href='https://app.reviewnb.com/Arcadia-Science/2024-paired-token-masking/pull/4/discussion/#comment-2568255084'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Review by keithchev — APPROVED** [review]
I just reviewed the main notebook on ReviewNB. I think it looks good! It felt a lot more compact and was easier/faster to read. And the quarto rendering looks great! (though I can't say I love the mini DSL to specify figure captions 🙈)

**Comment by ekiefl** [comment]
Thanks for your feedback @braebigge and @keithchev, I really appreciate the effort y'all have put in.

I incorporated all your feedback. Of particular note was highlighting the purpose of the pub early on in the introduction. I decided to create a summary section in the notebook pub template that offers the authors a box to write a summary / purpose / abstract depending on their needs. It looks like this:

image

For this pub, I added the following text:

> This notebook details a quick analysis on the ESM2 (Evolutionary Scale Modeling) model for masked token prediction. We stumbled upon a counter-intuitive result related to the effect that masking one residue has on the probability distribution of another. In brief, we noticed that that amino acid probability distributions of residues affect each other in a pattern that mirrors a protein's 3D contact map. However, this pattern diminishes for the larger model sizes. Our question to you is, *why*?

Second, to further highlight we're requesting community feedback, I added a callout in the Conclusion:

image
